browse-scaler/src/cmr.js

```diff
@@ -1,10 +1,32 @@
 const fetch = require('node-fetch');
+const { getSecureParam } = require('./util');
 
 /* CMR ENVIRONMENT VARIABLES */
 const cmrRootUrl = `http://${process.env.CMR_ROOT}`;
 const cmrCollectionUrl = `${cmrRootUrl}/search/collections.json?concept_id=`;
 const cmrGranuleUrl = `${cmrRootUrl}/search/granules.json?concept_id=`;
 
+/**
+ * getEchoToken: Fetch token for CMR requests
+ * @returns {String} ECHO Token for CMR requests
+ * @throws {Error} If no token is found. CMR will not return anything
+ * if no token is supplied.
+ */
+const getEchoToken = async () => {
+  console.log(process.env.CMR_ENVIRONMENT);
+  const response = await getSecureParam(
+    `/${process.env.CMR_ENVIRONMENT}/browse-scaler/CMR_ECHO_SYSTEM_TOKEN`
+  );
+
+  if (response === undefined) {
+    throw new Error('ECHO Token not found. Please update config!');
+  } else {
+    console.log('Retrieved ECHO TOKEN');
```

Comment:
Not sure if logging this every time the lambda is called is very useful. I think the error on failure would suffice.

Reply:
I have this here because I want to see the token retrieval happening.